Dockerfile

```diff
@@ -1,11 +1,21 @@
-FROM continuumio/miniconda3
-MAINTAINER Serenata de Amor "datasciencebr@gmail.com"
+FROM continuumio/miniconda3:4.5.11
+LABEL maintainer="Serenata de Amor <datasciencebr@gmail.com>"
```

Comment:
We don't use this email. Can we have `contato@serenata.ai` instead?

Reply:
Yes. Just a minute.